test/e2e/rolling_update_test.go

```diff
@@ -27,19 +27,19 @@ import (
 )
 
 const WAIT_DEPLOYMENT_AVAILABLE_TIMEOUT = time.Second * 180
-const OLD_VM_DELETION_TIMEOUT = time.Second * 60
+const OLD_VM_DELETION_TIMEOUT = time.Second * 30
 
 func doTestCaaDaemonsetRollingUpdate(t *testing.T, assert RollingUpdateAssert) {
 	runtimeClassName := "kata-remote"
 	namespace := envconf.RandomName("default", 7)
-	deploymentName := "nginx-deployment"
-	containerName := "nginx"
-	imageName := "nginx"
-	serviceName := "nginx-service"
+	deploymentName := "webserver-deployment"
+	containerName := "webserver"
+	imageName := "python:3"
```

Comment:
Does `python:3` support s390x also?

Reply:
Yes, it supports s390x: https://hub.docker.com/r/s390x/python/